socket_client.c

```diff
@@ -0,0 +1,87 @@
+/* uds_client.c */
+#include <sys/types.h>
+/* Pi-hole: A black hole for Internet advertisements
+*  (c) 2017 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  FTL Engine
+*  Unix socket connection test program
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license. */
```

Comment:
The license header should be at the top.

Reply:
Fixed by latest commit